OCExtensions/Classes/OCUtils.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  OCUtils.swift
+//  Pods
+//
+//  Created by Henrique Morbin on 03/08/16.
+//
+//
+
+import UIKit
+
+struct OCUtils {
```

Comment:
It is not a ViewController relative. This feature will be able to be called anywhere.